### ISSUE TITLE
Cannot click on ClearIndicator of SelectInput in RTL tests

### ISSUE THREAD

**Issue by ahmehri** [commercetools/ui-kit#1210]
**Describe the bug**
In RTL tests, clicking on the `ClearIndicator` of the `SelectInput` component has no effect.

**To Reproduce**

1. Write an RTL test that renders a `SelectInput` component with the default `ClearIndicator` component, i.e the X icon
2. get the `ClearIndicator` element
3. fire a click event on it
4. nothing happens

**Expected behavior**
The component should response to the fired event.

**Comment by jonnybel** [comment]
Thanks. I think we could make it easier to get the `ClearIndicator` by giving a title attribute to the SVG, and also by making its container actually a button instead of a div.

Meanwhile, see if you can make it work by following the example on this UI-kit test: https://github.com/commercetools/ui-kit/blob/master/src/components/inputs/select-input/select-input.spec.js#L320

**Comment by ahmehri** [comment]
I followed the example and was able to retrieve the `ClearIndicator` element. Clicking on it, however, didn't trigger the related callback.

**Comment by jonnybel** [comment]
OK, I've found the problem: `react-select` is passing `onMouseDown` and `onTouchEnd` handlers to the element, but not "onClick".
As such, I've tested and it does respond to `fireEvent.mouseDown(ClearIndicatorElement)`. Let me know if it works for you 🙂

Meanwhile, I'll open a PR to add a `label` and `title` to this button to make it easier to select for the tests, and I'll also update the `SelectInput` tests to illustrate the example.
